mkdocs.yml

```
@@ -21,6 +21,7 @@ nav:
   - Predictive Analytics:
     - Optimize fleet management: predictive-analytics/fleet_management/index.md
     - Optimize omni-channel inventory: predictive-analytics/inventory/index.md
+    - Next best offer for banking: predictive-analytics/next-best-offer/index.md
```

Hide in nav until PMM provides copy & graphics

```suggestion
    #- Next best offer for banking: predictive-analytics/next-best-offer/index.md
```